test/inductor/test_cudagraph_trees.py

```diff
@@ -1952,6 +1952,23 @@ def foo(x):
             ).run(captured_output[0])
             self.assertEqual(counters["inductor"]["cudagraph_skips"], 1)
 
+        @torch._dynamo.config.patch("capture_dynamic_output_shape_ops", True)
+        @torch._inductor.config.patch("cpp_wrapper", True)
+        def test_skip_cpp_wrapper(self):
+            def foo(x):
+                return x + 1
+
+            foo_c = torch.compile(mode="reduce-overhead")(foo)
+
+            with capture_stderr() as captured_output:
+                t = torch.rand([32], device="cuda")
+                self.assertEqual(foo(t), foo_c(t))
+
+            FileCheck().check("skipping cudagraphs due to cpp wrapper enabled").run(
```

Comment:
nit: error message could mention the `TORCHINDUCTOR_CPP_WRAPPER` env var to make it clearer how to address this

Reply:
imo we should refer to it as the config value, not the envvar.